README.md

```diff
@@ -74,21 +74,29 @@ Usually the message on the DONE queue will be processed by the captionWF in comm
 docker run --rm --tty --env-file .env sul-speech-to-text --receive
 ```
 
-## The Job File
+## The Job Message Structure
```

Comment:
"File" seemed like a remnant of the initial implementation where the job info was dropped on an S3 bucket, maybe?